## [Final Project - Create, Modify, and Submit a Jupyter Notebook](#shirali_project)



<a id='#shirali_project'></a>


#### A car with a full tank of gas drives 80 miles per hour for 3 hours before running out of gas. How many miles can the car go per tank?



In [5]:
# I would like to give by python code as following
print("The car go per tank is", 80*3,"miles")

The car go per tank is 240 miles


#### A runner travels a half marathon (13.1 miles) in 3 hours. How fast was this runner going in miles per hour?


In [6]:
print("The runner per our speed is", 13.1/3, "miles/h")

The runner per our speed is 4.366666666666666 miles/h


#### The spread of a virus is exponential. If 5 people are infected and each person infects 2 other people, how many people have been infected after 5 rounds of infection?

> The formula to calculate the infected people after 5 rounds is Total_infected = initial_infected * (infection_rate ** number_of_rounds)


In [20]:
initial_infected = 5
infection_rate = 2
number_of_rounds = 5

total_infected = initial_infected * (infection_rate ** number_of_rounds)

print("Total number of infected people after 5 rounds:", total_infected)


Total number of infected people after 5 rounds: 160


- [x] Answer for Q1 is 240miles;
- [x] Answer for Q2 is 4.36 miles/h;
- [x] Answer for Q3 is 160 total infected people after 5 rounds


| Questions | Answers |
| ---------------|----------------|
| Q1  | 240 miles |
| Q2  | 4.36 miles/h |
| Q3  | 160 people |


[Jump to my anchor](#shirali_project)

---

<a id='shirali_project'></a>
## My Anchor


I learnt a lot doing this project, thanks a lot Laura Harris